variables.tf

```diff
@@ -97,6 +97,12 @@ variable "key_pair_name" {
   description = "The name of the key-pair to associate with the SSM Agent instances. This can be (and probably should) left empty unless you specifically plan to use `AWS-StartSSHSession`."
 }
 
+variable "security_groups" {
```

Comment:
Can you update the name of this to `additional_security_group_ids`? I think that will more accurately reflect what this variable does and what values it needs.

Reply:
Hi @Gowiem, handled your remarks in [274a8ed](https://github.com/masterpointio/terraform-aws-ssm-agent/pull/4/commits/274a8ed91d1aa39e2e51f5997e419b9d7ade3c3b)